app/scripts/modules/amazon/serverGroup/details/serverGroupDetails.aws.controller.js

```diff
@@ -370,5 +373,11 @@ module.exports = angular.module('spinnaker.serverGroup.details.aws.controller',
       }
       return null;
     };
+
+    this.getAccountDetails = (serverGroup) => {
```

Comment:
`getAccountDetails` implies that this method returns the account details. Maybe a better name would be `addAccountDetails` or `addAccountDetailsToServerGroup`.